b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/idpSectionComponent.js

```diff
@@ -0,0 +1,446 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import AppSelectIcon from "@rsuite/icons/AppSelect";
+import React, { useCallback, useEffect, useState } from "react";
+import { Avatar, Button, Container, FlexboxGrid, Form, Modal, Stack, useToaster } from "rsuite";
+import Enterprise from "./data/templates/enterprise-identity-provider.json";
+import Google from "./data/templates/google.json";
+import IdentityProviderList from "./otherComponents/identityProviderList";
+import styles from "../../../../styles/idp.module.css";
+import decodeCreateIdentityProvider from
+    "../../../../util/apiDecode/settings/identityProvider/decodeCreateIdentityProvider";
+import decodeListAllIdentityProviders from
+    "../../../../util/apiDecode/settings/identityProvider/decodeListAllIdentityProviders";
+import { EMPTY_STRING, ENTERPRISE_ID, GOOGLE_ID, checkIfJSONisEmpty, sizeOfJson } from 
+    "../../../../util/util/common/common";
+import { errorTypeDialog, successTypeDialog } from "../../../common/dialog";
+import SettingsTitle from "../../../common/settingsTitle";
+
+/**
+ * 
+ * @param prop - session
+ * @returns The idp interface section.
+ */
+export default function IdpSectionComponent(prop) {
+
+    const { session } = prop;
+
+    const toaster = useToaster();
+
+    const [ idpList, setIdpList ] = useState([]);
+    const [ openAddModal, setOpenAddModal ] = useState(false);
+    const [ selectedTemplate, setSelectedTemplate ] = useState(undefined);
+
+    const templates = [
+        Enterprise,
+        Google
+    ];
+
+    useEffect(() => {
+        fetchAllIdPs();
+    }, [ fetchAllIdPs ]);
+
+    const fetchAllIdPs = useCallback(async () => {
+
+        const res = await decodeListAllIdentityProviders(session);
+
+        if (res) {
+            if (res.identityProviders) {
+                setIdpList(res.identityProviders);
+            } else {
+                setIdpList([]);
+            }
+        } else {
+            setIdpList(null);
+        }
+
+    }, [ session ]);
+
+    const onAddIdentityProviderClick = () => {
+        setOpenAddModal(true);
+    };
+
+    const onCreationDismiss = () => {
+        setSelectedTemplate(undefined);
+    };
+
+    const onIdpCreated = (response) => {
+        if (response) {
+            successTypeDialog(toaster, "Success", "Identity Provider Created Successfully");
+
+            setIdpList([
+                ...idpList,
+                response
+            ]);
+
+            setOpenAddModal(false);
+            setSelectedTemplate(undefined);
+        } else {
+            errorTypeDialog(toaster, "Error Occured", "Error occured while creating the identity provider. Try again.");
+        }
+    };
+
+    const onIdPSave = async (formValues, template) => {
+
+        if (checkEmpty(formValues, template)) {
+            errorTypeDialog(toaster, "Fields Cannot be Empty", "Form fields cannot be empty occured.");
```

Comment:
`Fields cannot be empty"`

Reply:
This was fixed when this file was moved to a different location

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/idpSectionComponent.tsx